From 3f1419ae1df11d2a7482350386d6065c3ced76d4 Mon Sep 17 00:00:00 2001
From: Charlie Paul <cpaul.windriver@gmail.com>
Date: Tue, 31 Mar 2015 16:50:48 -0700
Subject: [PATCH 054/213] misc: Changes made to support axxia BSP

These file are used to add the support for the misc
device on the LSI axxia 5500 board.

Signed-off-by: Charlie Paul <cpaul.windriver@gmail.com>
---
 drivers/misc/Makefile | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/misc/Makefile b/drivers/misc/Makefile
index 3198336..72e51c5 100644
--- a/drivers/misc/Makefile
+++ b/drivers/misc/Makefile
@@ -49,6 +49,9 @@ obj-$(CONFIG_LATTICE_ECP3_CONFIG)	+= lattice-ecp3-config.o
 obj-$(CONFIG_SRAM)		+= sram.o
 obj-y				+= mic/
 obj-$(CONFIG_GENWQE)		+= genwqe/
+obj-$(CONFIG_LSI_NCR)           += lsi-ncr.o
+obj-$(CONFIG_LSI_MTC)		+= lsi-mtc.o
+obj-$(CONFIG_LSI_SMMON)         += lsi-smmon.o
 obj-$(CONFIG_ECHO)		+= echo/
 obj-$(CONFIG_VEXPRESS_SYSCFG)	+= vexpress-syscfg.o
 obj-$(CONFIG_CXL_BASE)		+= cxl/
-- 
1.9.1

